packages/default/scss/pivotgrid/_layout.scss

```diff
@@ -205,3 +205,21 @@
     }
 
 }
+
+
+
+@include exports("pivotgrid/layout/ie11-fixes") {
+
+    // Pivotgrid is stretched beyond container in IE 11
+    // see https://github.com/telerik/kendo-theme-default/issues/647
+    .k-ie11 {
+        .k-pivot-layout {
+            width: 100%;
+            table-layout: fixed;
+        }
+        .k-pivot-layout > tbody > tr > td:first-child {
+            width: 280px;
```

Comment:
Oddly specific 🙂

Reply:
I know, but it was the only I thought of